recipes/darmonpoints/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "darmonpoints" %}
+{% set version = "7.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b827ad80527d3356bef7b7c2480d73e7a060a8f4c183729990de0b166625c6bf
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win or py < 39]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - fplll
+    - fpylll
+    - gap-defaults
+    - matplotlib-base
+    - sagelib
+    - sagemath-db-conway-polynomials
+    - sagemath-db-elliptic-curves
+    - zn_poly
+
+test:
+  imports:
+    - darmonpoints.mixed_extension
+  requires:
+    - pip
+  commands:
+    - pip check
```

Comment:
```suggestion
  # pip check is known to be broken for SageMath packages because sagemath-standard relies on too many dependencies that are not required by sagelib.
  # requires:
  #   - pip
  # commands:
   #  - pip check
```